homeassistant/components/switch/fritzdect.py

```diff
@@ -120,17 +121,65 @@ def is_on(self):
 
     def turn_on(self, **kwargs):
         """Turn the switch on."""
-        actor = self.data.fritz.get_actor_by_ain(self.data.ain)
-        actor.switch_on()
+        from requests.exceptions import RequestException, HTTPError
```

Comment:
requests are a core requirement and should be imported on top of the script

Reply:
Fixed